handlers.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type HandlersEnv struct {
+	service *Service
+}
+
+func (h *HandlersEnv) indexHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "hello")
+}
+
+func (h *HandlersEnv) devicesHandler(w http.ResponseWriter, r *http.Request) {
+	var devPayload DevicePayload
+
+	err := json.NewDecoder(r.Body).Decode(&devPayload)
+	if err != nil {
+		fmt.Println(fmt.Errorf("handlerError: %v", err))
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	_, err = h.service.AddDevice(&devPayload)
+	if err != nil {
+		fmt.Println(fmt.Errorf("handlerError: %v", err))
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	d := h.service.tempDevice
```

Comment:
I've only implemented that because we decided to return the id instead of an object, and that is the next task yet ahead of me. After I implement that I won't need it at all, it's only here for the tests, so i can easily check if the device was created.

Reply:
I think we misunderstood each other. I meant returning id only in DAO. The service should return Device

Reply:
I've read through to the end, sorry, indeed I haven't understood you :)